recipe/meta.yaml

```diff
@@ -47,12 +46,15 @@ test:
     - nuitka.utils
   requires:
     - pip
+    - {{ compiler('c') }}  # [linux]
   commands:
     - pip check
     - nuitka --help
+    - nuitka-run --help
+    - python -m nuitka --version
```

Comment:
There is also a test suite upstream. What about running this?

Reply:
It's not working because of a frozen prompt without choosing a variant Yes/no.

Reply:
The test suite has a pretty complicated settings https://github.com/Nuitka/Nuitka/blob/1.5.8/nuitka/tools/testing/run_nuitka_tests/__main__.py

Reply:
yeah at first glance it looked pretty complicated/non-automatable. No worries, we can leave it out.

Reply:
I've fixed the test compilation by disabling ccache in `nuitka --show-scons --run --verbose --disable-ccache Empty.py` it bypassed the downloading step

Reply:
Is it not possible to automate the test suite by wrapping the suite in a script that is able to choose variant Yes/No?